scripts/header.sh

```diff
@@ -2,7 +2,7 @@
 
 ORG_ID="$1"
 USER_NAME="$2"
```

Comment:
What about a default user for this script, so that I can put that in my config.yaml as a user with access  and forget about it?